src/build/arg/mod.rs

```diff
@@ -846,6 +846,10 @@ impl<'help> Arg<'help> {
     /// (i.e. if A conflicts with B, defining A.conflicts_with(B) is sufficient. You do not
     /// need to also do B.conflicts_with(A))
     ///
+    /// **NOTE:** When using a YAML configuration file, this option can take both a single string
+    /// and an array of strings. Trying to use [`Arg::conflicts_with_all`] in this environment will
```

Comment:
Let's not write the second sentence

Reply:
In fact, let's not mention anything here, since we clearly mention it during `conflicts_with_all`.

Reply:
I haven't opened many PRs, but isn't "raise an error" missing?

Reply:
I don't understand what you are asking. We don't raise an error when people are using `conflicts_with`. We raise when they are using `conflicts_with_all`

Reply:
That's what I wrote ("Trying to use [`Arg::conflicts_with_all`] in this environment will raise an error")

Reply:
And you wrote it in the documentation for `Arg::conflicts_with` which is completely not needed since we are mentioning this again in `Arg::conflcits_with_all`

Reply:
Okay, sorry. I think I misunderstood what you meant in the first comment. My bad